src/backend/bundler/config.js

```diff
@@ -44,9 +41,6 @@ const globals = {
   axios: 'axios',
   flat: 'flat',
   recharts: 'Recharts',
-  'react-select': 'ReactSelect',
-  'react-select/async': 'ReactSelectAsync',
-  'react-select/creatable': 'ReactSelectCreatable',
```

Comment:
Are you sure we don't need these?

Reply:
Moved to globals in design system https://github.com/SoftwareBrothers/adminjs-design-system/pull/39/commits/0ee3ceeed3d34ad608526a2f3a924d827fdbd6c2